### ISSUE TITLE
[SUGGESTION] Model Placing

### ISSUE THREAD

**Issue by SebyRo23** [oraxen/oraxen#553]
### I have checked that...

- [X] ...such a feature does not exist already
- [X] ...such a feature request has not been submitted already

### Description

The model placement feature can be used to set diffrent models based on which side of a block the furniture/noteblock/stringblock on
Config:

  Mechanics:
    stringblock:
      model_placement:
        north_block: paper_lantern_north
        east_block: paper_lantern_east
        south_block: paper_lantern_south
        west_block: paper_lantern_west
        
How it would look in game
https://ibb.co/bgFDShF

### Reason

Some Players wish to recreate the behavior of some blocks(lanterns, bells)

**Comment by Boy0000** [comment]
this is already implemented through [directional block](https://docs.oraxen.com/mechanics/block-mechanic/directional-mechanic) mechanic...

**Comment by SebyRo23** [comment]
Its not like directional blocks you cant achive model placement using them

my suggestions is about having diffrent models based on what side of the block is the furniture is present on

**Comment by Boy0000** [comment]
You are saying furniture but your example has a stringblock.
Also you can use directional blocks to make x blockface place y block... thats the whole point

Tldr this wont be added as I believe you are asking for what directional blocks already does

**Comment by SebyRo23** [comment]
understood now i have to make a bug report on string blocks not working with directional blocks

**Comment by Boy0000** [comment]
Because stringblocks arent aimed to be directional indeed. Only one that is meant to be is noteblock mechanic. So it isnt a bug

**Comment by SebyRo23** [comment]
ohh ok but are there any plans to add it?

**Comment by Boy0000** [comment]
Not atm no

**Comment by SebyRo23** [comment]
ok but i make a suggestion?
